README.md

```diff
@@ -1 +1,26 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# **Jessica Lee** (*preferred*: Jess)
+
+![This is me!](/Users/JessicaLee/Desktop/Jess.jpg)
+
+## **About Me**
+
+* I am from Toronto, Canada. 
+* Things I like to do when I am not academically stressed:
+   + Yoga
+   + Cook & Eat
+      - [my food instagram](https://www.instagram.com/p.s.ilovefoood/)
+   + Read
+   + Build LEGOs
+   + Travel
+  
+## **Research**
+
+* I am a MAPSS student concentrating in Psychology.
+* With my MA thesis, I am investigating whether race is a salient cue for preverbal infants when forming expectations about third-party interracial interactions. [Infant Learning and Development Laboratory](https://voices.uchicago.edu/woodwardlab/)
+* General research interests
+   + Development of social categories in infants and children
+   + Intergroup relations and discrimination
+   + How diversity in social envioronments influence the above concepts
+ 
```

Comment:
Overall, clean code! 
Things I liked:
- Relevant use of links (you are a part of really cool lab!)
- Relevant use of lists 

Things that could be improved on:
- The image of you was not able to show up in the preview :-( I hope the comment I made above helps you out!
- Having slightly more concise commit messages would be helpful in knowing what changes you have made